smallrye-reactive-messaging-provider/src/main/java/io/smallrye/reactive/messaging/extension/LegacyEmitterImpl.java

```diff
@@ -41,7 +43,7 @@ public boolean isRequested() {
     @Override
     public synchronized Emitter<T> send(T msg) {
         if (msg == null) {
-            throw new IllegalArgumentException("`null` is not a valid value");
+            throw ex.illegalArgumentForValue("null");
```

Comment:
do we have to pass "null" ?

Reply:
Maybe change the message to be about null to being valid and use a specific method name for it?

Reply:
I reuse that for other fields (I think)

Reply:
Replace with dedicated field.